### PR TITLE
Two small changes

### PR DESCRIPTION
**Pull request by HinTak**
Mainly it is the face->foreach_glyph method that I am contributing...

### CODE REVIEW

File: FreeType.xs [hunk 1]
```diff
@@ -144,6 +144,9 @@ const static QefFT2_Uv_Const qefft2_uv_const[] =
     QEFFT2_CONSTANT(FT_LOAD_LINEAR_DESIGN)
 #if QEFFT2_FT_AT_LEAST(2,1,3)
     QEFFT2_CONSTANT(FT_LOAD_NO_AUTOHINT)
+#endif
+#if QEFFT2_FT_AT_LEAST(2,6,1)
+    QEFFT2_CONSTANT(FT_LOAD_COMPUTE_METRICS)
```
Comment (basiliscos) [inline FreeType.xs]: If u export constant it needs to be documented/mentioned in `Freetype.pm`, as the other constants.

**Comment by HinTak** [comment]
Okay - I'll push another commit for that. Thanks.
    On Wednesday, 8 August 2018, 00:50:34 GMT+8, Ivan Baidakou <notifications@github.com> wrote:  
 
 
@basiliscos commented on this pull request.

In FreeType.xs:
> @@ -144,6 +144,9 @@ const static QefFT2_Uv_Const qefft2_uv_const[] =
     QEFFT2_CONSTANT(FT_LOAD_LINEAR_DESIGN)
 #if QEFFT2_FT_AT_LEAST(2,1,3)
     QEFFT2_CONSTANT(FT_LOAD_NO_AUTOHINT)
+#endif
+#if QEFFT2_FT_AT_LEAST(2,6,1)
+    QEFFT2_CONSTANT(FT_LOAD_COMPUTE_METRICS)

If u export constant it needs to be documented/mentioned in Freetype.pm, as the other constants.

**Comment by basiliscos** [comment]
Please, add some simple test into `t/` , and update Changes. I'll merge and release to cpan then.

**Comment by HinTak** [comment]
The compute metric change (I sent upstream as well as the C# and python bindings) is for verifying / calculating HDMX/etc metric tables - for well behaving fonts, with or without it should be numerically identical (subjected to small differences in changes in FreeType'a rasterization from time to time) so there isn't really a simple test. 
I 'll add a test for face->foreach_glyph along the same line as foreach_characters .

**Comment by HinTak** [comment]
foreach_glyph on bdf is not really interesting - FreeType seems to add a glyph 0 internally , but otherwise the same. So I only added one on Vera, and also did a bit of renaming/editing of the foreach_char test for correctness.

**Comment by basiliscos** [comment]
Thank you for your contribution!

**Pull request merged**